eslint.config.mjs

```diff
@@ -8,8 +8,15 @@ export default tseslint.config(
     ...tseslint.configs.recommended,
     {
         rules: {
+            // Unused vars reported as error
+            "@typescript-eslint/no-unused-vars": [
+                "error",
+                {
+                    // Do not report unused function arguments
+                    "args": "none",
```

Comment:
I was wondering why we wouldn't enforce the rule to error out when unused arguments are declared. We can force the contributor to remove tech debt by having this rule on and minimize the time for this when reviewing PRs.  I would be keen to hear your thoughts :)

Reply:
The problem I have found is that the error will raise as well in a situation like these:

```js
 expressCustomData(error: Error, request: Request) {
    return {};
  }
```

Here, we define a function that the user can override, but the default implementation doesn't use its parameters, and so `eslint` would complain about that.

In the README, you can see an example use of this, where the two parameters are defined:

```js
raygunClient.expressCustomData = function (err, req) {
  return { 'level': err.level };
};
```

Alternatively, we can add this rule and then add the "ignore" comment on each one of the individual cases where it should be.

Reply:
I see now. So this is in regard to the function needed to match the signature specified so we can't just use `expressCustomData ` without the parameters. To my understanding.

I'm happy with this justification :) We will just need to keep an eye on code reviews to ensure we aren't leaving being unused parameters that aren't needed

Reply:
Yes, if we see that this becomes a problem, I'll look for an alternative.